.github/repo_policies/BOT_APPROVED_FILES

```diff
@@ -8,5 +8,4 @@ ic-os/hostos/context/docker-base.prod
 ic-os/setupos/context/docker-base.dev
 ic-os/setupos/context/docker-base.prod
 mainnet-canisters.json
-testnet/mainnet_revisions.json
-mainnet-canisters.json
+mainnet_revisions.json
```

Comment:
The name `mainnet_revisions.json` is a bit ambiguous since it could refer to revisions of subnets or of canisters on mainnet. Why not make the name more explicit like:

`mainnet-subnet-revisions.json`

Likewise you could consider renaming `mainnet-canisters.json` to something consistent with the previous like:

`mainnet-canister-revisions.json`